projects/schematics/src/migrations/4_0/constructor-deprecations/data/guest-register-form.component.migration.ts

```diff
@@ -0,0 +1,22 @@
+import { SPARTACUS_CHECKOUT_COMPONENTS } from './../../../../shared/constants';
+import {
+  GUEST_REGISTER_FORM_COMPONENT,
+  SPARTACUS_CORE,
+  SPARTACUS_USER_PROFILE_ROOT,
+  USER_REGISTER_FACADE,
+  USER_SERVICE,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const GUEST_REGISTER_FORM_COMPONENT_MIGRATION: ConstructorDeprecation = {
+  // feature-libs/checkout/components/order-confirmation/components/guest-register-form/guest-register-form.component.ts
+  class: GUEST_REGISTER_FORM_COMPONENT,
+  importPath: SPARTACUS_CHECKOUT_COMPONENTS,
+  deprecatedParams: [{ className: USER_SERVICE, importPath: SPARTACUS_CORE }],
```

Comment:
In deprecated params we need to describe all params from the previous constructor. In this case, as the changed service was first, we need to remove all params and then add all params again in this migration.

Also add info about this contructor change in the doc

Reply:
Added.